multinode-demo/common.sh

```diff
@@ -88,16 +88,16 @@ tune_system() {
       # test the existence of the sysctls before trying to set them
       # go ahead and return true and don't exit if these calls fail
       sysctl net.core.rmem_max 2>/dev/null 1>/dev/null &&
-          sudo sysctl -w net.core.rmem_max=1610612736 1>/dev/null 2>/dev/null
+          sudo sysctl -w net.core.rmem_max=161061273 1>/dev/null 2>/dev/null
```

Comment:
@sakridge, what's the significance of this magic number? Just dropping the least significant digit seems weird.

Reply:
only significance is that it works a lot better than the other number.